.pipelines/e2e-step-template.yaml

```diff
@@ -20,7 +20,7 @@ steps:
       export CLIENT_ID=$(SERVICE_PRINCIPAL_CLIENT_ID_E2E_KUBERNETES)
       export CLIENT_SECRET=$(SERVICE_PRINCIPAL_CLIENT_SECRET_E2E_KUBERNETES)
       export CLIENT_OBJECTID=$(SERVICE_PRINCIPAL_OBJECT_ID_E2E_KUBERNETES)
-      make test-kubernetes
+      bash -c 'eval `ssh-agent` && make test-kubernetes'
```

Comment:
do we need to add this to Jenkins tests as well? What about running tests locally? Should it just be part of the make target?

Reply:
So this actually works. (which is an improvement) Next step is to see if we can do one of your alternate, preferable proposals.